CMakeLists.txt

```diff
@@ -1,29 +1,40 @@
 cmake_minimum_required(VERSION 3.15)
 
-project(HyKKT VERSION "0.0.1")
+# Adds version settings and set variable CMAKE_PROJECT_VERSION
+project(HyKKT VERSION "0.1.0")
 
 set(CMAKE_CXX_STANDARD 11)
 
+set(PACKAGE_NAME  "HyKKT")
+set(PACKAGE_TARNAME "hykkt")
+
 option(HYKKT_TEST_WITH_BSUB "Use `jsrun` instead of `mpirun` commands when running tests" OFF)
 set(HYKKT_CTEST_OUTPUT_DIR ${PROJECT_BINARY_DIR} CACHE PATH "Directory where CTest outputs are saved")
 
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}")
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+
+# For Unix CMAKE_INSTALL_PREFIX defaults to /usr/local
+# If user doesn't have permission to write to /usr/local than the below line is recommended
+# set(CMAKE_INSTALL_PREFIX "${CMAKE_SOURCE_DIR}/install" CACHE PATH "${CMAKE_SOURCE_DIR}/install"FORCE)
+
+message(STATUS "Installation Destination is ${CMAKE_INSTALL_PREFIX}")
+
+# Including clang-format cmake files to do automatic checking of formating
+include(./cmake/clang-format)
```

Comment:
I think this relative path should probably be changed to an absolute path using the right CMake variables, or we should configure a module path so that this is a little cleaner for cuda and clang-format utilities (maybe we can take care of this in the linting PR) - maybe there should be an issue for the linting PR on GitHub?

Reply:
Will be addressed in #6